content/SCALE/SCALEUIReference/SystemSettings/Services/S3ServiceScreen.md

```diff
@@ -13,98 +13,20 @@ tags:
-
-**Correct Example**
-```
-host_base = `192.168.123.207:9000`
-host_bucket = `192.168.123.207:9000`
-```
-
-**Incorrect Example**
-```
-host_base = `192.168.123.207`
-host_bucket = `%(bucket)s.192.168.123.207`
-```
-
-Poll the buckets using `s3cmd ls` to see the buckets created with the Minio browser.
-
-For more information on using Minio with `s3cmd`, see https://docs.minio.io/docs/s3cmd-with-minio.html and https://s3tools.org/s3cmd.
-
-### S3 Browser (Windows)
-
-The Windows PC S3 browser is another convenient way to connect to the Minio S3 from TrueNAS.
-
-To set it up, first [install the S3 browser](https://s3-browser.en.uptodown.com/windows).
-
-After installation completes, add a new account. 
-
-![AmazonS3NewAccount](/images/CORE/AmazonS3NewAccount.png "S3 Browser: New Account")
-
-In the settings, select **S3 Compatible Storage** as the **Account Type**, then enter the Minio access point similar to the `s3cmd` setup (TrueNAS_IP_address:9000 or other port if set differently).
-Select the SSL settings appropriate for the particular setup.
-The S3 browser assumes SSL by default, but it can be unset for a LAN attached session.
-
-![AmazonS3EditAccount](/images/CORE/AmazonS3EditAccount.png)
-
-It is possible to access, create new buckets, or upload files to created buckets.
-
-![AmazonS3Browser](/images/CORE/AmazonS3Browser.png "S3 Browser")
+![S3ServiceSettings(/images/SCALE/22.02/S3ServiceSettings.png "S3 Service Options")
 
+| Settings | Description |
+|----------|-------------|
+| **IP Address** | Select an IP address from the dropdown list options **0.0.0.0**, **::**, or to enter the IP address that runs the S3 service. Select **0.0.0.0** to tell the server to listen on all addresses. Select the TrueNAS IP address to constrain it to a specific network. |
+| **Port** | Enter the TCP port that provides the S3 service.   |
+| **Console Port** | Enter a static port for the MinIO web console. Default is 9001. |
+| **Access Key** | Enter the S3 access ID. See [Access keys](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) for more information. |
+| **Secret Key** | Enter the S3 secret access key. See [Access keys](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) for more information. |
+| **Disk** | Enter or use <iconify-icon icon="bxs:right-arrow"></iconify-icon> to the left of <span class="material-icons">folder</span> **/mnt** to browse to a directory to define the S3 file system path. |
+| **Enable Browser** | Enables the S3 service web UI. Access the MinIO web UI by entering the IP address and port number separated by a colon in the browser address bar. Example: *192.168.1.0:9000*. |
+| **Certificate** | Use an SSL [certificate]({{< relref "CertificatesSCALE.md" >}}) created or imported in **Credentials > Certificates** for secure S3 connections. |
```

Comment:
Build test found this reference is ambiguous. It'll need to be adjusted with the path to the desired file.
![image](https://user-images.githubusercontent.com/17933320/191568186-1a93cb72-afb1-4bc1-8ab9-c32d1d164292.png)

```suggestion
| **Certificate** | Use an SSL [certificate]({{< relref "CertificatesSCALE.md" >}}) created or imported in **Credentials > Certificates** for secure S3 connections. |
```